Initium-Odp/war/WEB-INF/odppages/ajax_group.jsp

```diff
@@ -11,7 +11,7 @@
 			<c:forEach var="groups" items="${warDecGroupNames}">
 				<p>${$groups}</p>
 				<br/>
-				<a onclick='endWar(event, "<c:out value="${groups}"/>")'>End War</a>
+				<a onclick='endWar(event, ${groups})'>End War</a>

```

Comment:
This change seems wrong. No quotes around the string argument mostly. Also using c out will escape characters for you.